lib_sample/schema/baguettes.ex

```diff
@@ -10,4 +10,6 @@ defmodule EctoCrux.Schema.Baguettes do
   """
 
   use EctoCrux, module: EctoCrux.Schema.Baguette
+
+  require EctoCrux.Repo
```

Comment:
So all crux users will have to use and require ? :/

Reply:
Non, c'est simplement pour s'assurer que `EctoCrux.Repo` soit compilé avant `EctoCrux.Schema.Baguettes`. C'est donc spécifique à l'exemple Baguettes intégré à Crux. Sans cette ligne, la compilation de `EctoCrux.Repo` intervient trop tard puisque ici nous ne sommes pas réellement dans le contexte global d'un projet Phoenix.